src/Bart/Git/Commit.php

```diff
@@ -57,6 +57,21 @@ public function message()
 		return $result->getOutput(true);
 	}
 
+	/** 
+	 * @return string List of modified files
+	 */
+
```

Comment:
Please remove this extraneous line.

Is the return type string[] or string? We should be returning an array of strings, one for each file path. 

```
/**
 * @return string[] The files names changed in commit
 */